workshops/models.py

```diff
@@ -391,6 +490,206 @@ def save(self, *args, **kwargs):
+        choices=DATA_TYPES_CHOICES,
+        verbose_name='We currently have developed or are developing workshops'
+                     ' focused on four types of data. Please let us know which'
+                     ' workshop would best suit your needs.',
+        blank=True,
+    )
+    data_types_other = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Other data domains for the workshop',
+        blank=True,
+    )
+    attendee_academic_levels = models.ManyToManyField(
+        'AcademicLevel',
+        help_text='If you know the academic level(s) of your attendees, '
+        'indicate them here.',
+        verbose_name='Attendees\' Academic Level',
+    )
+    attendee_computing_levels = models.ManyToManyField(
+        'ComputingExperienceLevel',
+        help_text='Indicate the attendees\' level of computing experience, if '
+        'known. We will ask attendees to fill in a skills survey before the '
+        'workshop, so this answer can be an approximation.',
+        verbose_name='Attendees\' level of computing experience',
+    )
+    attendee_data_analysis_level = models.ManyToManyField(
+        'DataAnalysisLevel',
+        help_text='If you know, indicate learner\'s general level of data '
+                  'analysis experience',
+        verbose_name='Level of data analysis experience',
+    )
+    cover_travel_accomodation = models.BooleanField(
+        default=False,
+        verbose_name='My institution will cover instructors\' travel and '
+        'accommodation costs.',
+    )
+    understand_admin_fee = models.BooleanField(
+        default=False,
+        verbose_name='I understand the Software Carpentry Foundation\'s '
+        'administration fee.',
+        help_text='<a href="http://software-carpentry.org/blog/2015/07/changes'
+        '-to-admin-fee.html">Look up administration fees</a>',
+    )
+    fee_waiver_request = models.BooleanField(
+        help_text='Waiver\'s of the administrative fee are available on '
+        'a needs basis. If you are interested in submitting a waiver '
+        'application please indicate here.',
+        verbose_name='I would like to submit an administrative fee waiver '
+        'application',
+        default=False,
+    )
+    TRAVEL_REIMBURSEMENT_CHOICES = (
+        ('', 'Don\'t know yet.'),
+        ('book', 'Book travel through our university or program.'),
+        ('reimburse', 'Book their own travel and be reimbursed.'),
+        ('', 'Other (type below)'),
+    )
+    travel_reimbursement = models.CharField(
+        max_length=STR_MED,
+        verbose_name='For instructor travel, how will instructors be'
+                     ' reimbursed?',
+        choices=TRAVEL_REIMBURSEMENT_CHOICES,
+        null=True, blank=True, default=None,
+    )
+    travel_reimbursement_other = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Other type of reimbursement',
+        blank=True,
+    )
+
+    ADMIN_FEE_PAYMENT_CHOICES = (
```

Comment:
Fees have changed - please see http://software-carpentry.org/workshops/request.html for current options.